Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -361,40 +366,6 @@ void GcodeSuite::G28(const bool always_home_all) {
 
   #endif // !DELTA (G28)
 
-  /**
-   * Preserve DXC mode across a G28 for IDEX printers in DXC_DUPLICATION_MODE.
-   * This is important because it lets a user use the LCD Panel to set an IDEX Duplication mode, and
-   * then print a standard GCode file that contains a single print that does a G28 and has no other
-   * IDEX specific commands in it.
-   */
-  #if ENABLED(DUAL_X_CARRIAGE)
-
-    if (dxc_is_duplicating()) {
-
-      // Always home the 2nd (right) extruder first
-      active_extruder = 1;
-      homeaxis(X_AXIS);
-
-      // Remember this extruder's position for later tool change
-      inactive_extruder_x_pos = current_position[X_AXIS];
-
-      // Home the 1st (left) extruder
-      active_extruder = 0;
-      homeaxis(X_AXIS);
-
-      // Consider the active extruder to be parked
-      COPY(raised_parked_position, current_position);
-      delayed_move_time = 0;
-      active_extruder_parked = true;
-      extruder_duplication_enabled = IDEX_saved_duplication_state;
-      extruder_duplication_enabled = false;
-
-      dual_x_carriage_mode         = IDEX_saved_mode;
-      stepper.set_directions();
-    }
-
-  #endif // DUAL_X_CARRIAGE
-
```

Comment:
This block here was added by @Roxy-3D in #11690 so be sure to confer about it.

Reply:
Just did on discord. At least as it sits, it left my machine in an inconsistent state doing this alone. I asked Roxy to review this and let me know her thoughts when she can. The block here is also essentially a duplicate of a block above with an extra couple lines. Initially I combined them, then modified to ensure a consistent state post homing every time.